### PR TITLE
Add the health tool page

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the health tool page.
> Fixes #

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: swan-lake/integration-technology-tools/health-tool.md [hunk 1]
```diff
@@ -3,7 +3,198 @@ layout: ballerina-health-support-left-nav-pages-swanlake
 title: Health tool (FHIR/HL7)
 description: The health tool generates accelerators for developing healthcare integrations in Ballerina.
 keywords: ballerina, programming language, health, contract
-permalink: /learn/edi-tool/
+permalink: /learn/health-tool/
 active: health-tool
 intro: The health tool generates accelerators for developing healthcare integrations in Ballerina.
 ---
+
+The Health tool can generate accelerators primarily focussed on the [Fast Healthcare Interoperability Resources (FHIR) standard](https://www.hl7.org/fhir/overview.html). The Health tool is supported from Ballerina version 2201.7.0 (Swan Lake Update 7) onwards.
```
Comment (anupama-pathirage) [inline swan-lake/integration-technology-tools/health-tool.md:11]: remove the `standard` word from link text. i.e. this should be `[Fast Healthcare Interoperability Resources (FHIR)](https://www.hl7.org/fhir/overview.html) standard`

Comment (anupama-pathirage) [inline swan-lake/integration-technology-tools/health-tool.md:11]: Swan Lake part is missing. i.e. this should be `supported from Ballerina Swan Lake version 2201.7.0`

File: swan-lake/integration-technology-tools/health-tool.md [hunk 2]
```diff
@@ -3,7 +3,198 @@ layout: ballerina-health-support-left-nav-pages-swanlake
...
+
+## Ballerina FHIR package generator
+
+This tool is available in the latest version of the Health tool (v1.0.0). It generates a Ballerina package for a given FHIR implementation guide. The FHIR resources in the implementation guide will be represented as Ballerina records including the correct cardinality constraints and metadata.
```
Comment (anupama-pathirage) [inline swan-lake/integration-technology-tools/health-tool.md:15]: This sentence seems not needed. `This tool is available in the latest version of the Health tool (v1.0.0).`

File: swan-lake/integration-technology-tools/health-tool.md [hunk 3]
```diff
@@ -3,7 +3,198 @@ layout: ballerina-health-support-left-nav-pages-swanlake
...
+
+FHIR integration developers can leverage the generated package when transforming custom health data into FHIR format without referring to the specification.
+
+## Prerequisites
+
+- [Ballerina 2201.7.0 (Swan Lake Update 7)](https://ballerina.io/downloads/)
+- A directory containing all FHIR specification definition files.
+
+    >**Tip:** You can download a preferred standard implementation guide from the [FHIR Implementation Guide registry](http://fhir.org/guides/registry/).
```
Comment (anupama-pathirage) [inline swan-lake/integration-technology-tools/health-tool.md:24]: It is not that good to have `Tip` and `Note` both together. Seems tip line can be merged to `A directory containing all FHIR specification definition files.` it self.

File: swan-lake/integration-technology-tools/health-tool.md [hunk 4]
```diff
@@ -3,7 +3,198 @@ layout: ballerina-health-support-left-nav-pages-swanlake
...
+    Running executable
+    ```
+
+8. Invoke the API to test it.
+
+    ```
+    $ curl --location --request GET 'http://localhost:9090/Patient/2121'
+    ```
+
+    You view the response shown below.
```
Comment (anupama-pathirage) [inline swan-lake/integration-technology-tools/health-tool.md:172]: Should this be `You can view ...`

File: swan-lake/integration-technology-tools/health-tool.md [hunk 5]
```diff
@@ -3,7 +3,198 @@ layout: ballerina-health-support-left-nav-pages-swanlake
...
+
+    ```
+    {
+    "resourceType" : "Patient",
```
Comment (anupama-pathirage) [inline swan-lake/integration-technology-tools/health-tool.md:176]: Seems this is not properly indented.

**Comment by isuruuy429** [comment]
Shall we do the following changes?
 
1. Links to open in a new tab
2. In the **Example** section, Step 2-> '**Carin BB implementation guide**' link does not work

**Comment by isuruh15** [comment]
Let's add [http://hl7.org/fhir/us/carin-bb/STU2/downloads.html](http://hl7.org/fhir/us/carin-bb/STU2/downloads.html) as the link.
In this case we may need to alter the text as the linked page contains multiple files that can be downloaded.

so the step 2 can be modified as; 
Download the [Carin BB implementation guide](http://hl7.org/fhir/us/carin-bb/STU2/definitions.json.zip) and extract it to your current directory. -> 
Download the `JSON Definitions` zip archive from the [Carin BB implementation guide](http://hl7.org/fhir/us/carin-bb/STU2/downloads.html) and extract it to your current directory.

File: swan-lake/integration-technology-tools/health-tool.md (hunk 1 above)
Comment (praneesha) [reply swan-lake/integration-technology-tools/health-tool.md:11]: Fixed.

Comment (praneesha) [reply swan-lake/integration-technology-tools/health-tool.md:11]: Updated.

File: swan-lake/integration-technology-tools/health-tool.md (hunk 2 above)
Comment (praneesha) [reply swan-lake/integration-technology-tools/health-tool.md:15]: Removed.

File: swan-lake/integration-technology-tools/health-tool.md (hunk 3 above)
Comment (praneesha) [reply swan-lake/integration-technology-tools/health-tool.md:24]: Merged.

File: swan-lake/integration-technology-tools/health-tool.md (hunk 4 above)
Comment (praneesha) [reply swan-lake/integration-technology-tools/health-tool.md:172]: Updated.

File: swan-lake/integration-technology-tools/health-tool.md (hunk 5 above)
Comment (praneesha) [reply swan-lake/integration-technology-tools/health-tool.md:176]: Fixed.

**Comment by praneesha** [comment]
The above review comments are addressed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7556.

**Pull request merged**